packages/sdk-metrics/src/view/View.ts

```diff
@@ -182,6 +192,13 @@ export class View {
    * @param viewOptions.meterSchemaUrl
    * Instrument selection criteria:
    *  The schema URL of the Meter. No wildcard support, schema URL must match exactly.
+   * @param viewOptions.aggregationCardinalityLimit
```

Comment:
nit: I think it'd be good to move that entry up to the other stream altering options.